src/test/java/de/mediathekview/mserver/crawler/srf/parser/SrfFilmJsonDeserializerTest.java

```diff
@@ -0,0 +1,122 @@
+package de.mediathekview.mserver.crawler.srf.parser;
+
+import com.google.gson.JsonElement;
+import de.mediathekview.mlib.daten.Film;
+import de.mediathekview.mlib.daten.Resolution;
+import de.mediathekview.mlib.daten.Sender;
+import de.mediathekview.mserver.crawler.srf.tasks.SrfTaskTestBase;
+import de.mediathekview.mserver.testhelper.JsonFileReader;
+import java.net.URL;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Optional;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Parameterized.class)
+public class SrfFilmJsonDeserializerTest extends SrfTaskTestBase {
+
+  @Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(new Object[][] { 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)